NewEvaluation.css

```diff
@@ -180,6 +148,52 @@ h3{
 
 }
 
+/*---------------------- TextArea Box---------------------------*/
+
+.user-textarea textarea{
+	width:100%;
+	height: 100px;
+}
+
+
+
+/*------------------- Legend Box ------------------*/
+.legend-box legend{
+	color:#03a8f3;
+	font-size: 18px;
+	font-weight: 500;
+	padding-left: 1em;
+	padding-right: 1em;
+}
+
+.legend-box li{
+	float:left;
+	width: 33%;
```

Comment:
There's `calc()` and there's `box-sizing`. In your case, you probably want to look at the latter before anything else, though the former may come in handy every now and then, too.